best_practices.md

```diff
@@ -0,0 +1,233 @@
+- use easy to understand variable names
+- insert documentation or comments in your code that explain what is going on
+
+This kind of organization increases readability of your code, thereby reducing
+the risk of mistakes and making your code accessible to others.
+
+Style guides typically come with automatic formatters, that can automatically
+format your code according to the chosen style.
+In addition to automatic formatters, tools for checking adherence to a style
+guide as well as for common programming mistakes are usually also available.
+These tools are called linters.
+It is recommended that you make use of an automatic formatter and linter.
+For example, a popular automatic formatter for Python is
+[black](https://black.readthedocs.io), while a good linter for Python is
+[pylint](https://pylint.org/).
+It is recommended that you use an [online service to host your code](#host-your-code-on-a-website-with-an-issue-tracker),
+so you can [automate checking the quality of your code](#use-a-code-quality-checker)
+and even [formatting it](https://pre-commit.ci), should you wish to do so.
+
+## Keep your code reliable and maintainable
+If you write large pieces of code, it is recommended that you implement some
+form of
+[automated testing](https://the-turing-way.netlify.app/reproducible-research/testing.html)
+to ensure that you do not loose more and more time testing that your code still
+produces the correct results as it grows.
+This can range from
+[unit tests](https://the-turing-way.netlify.app/reproducible-research/testing/testing-unittest.html),
+that test each function in your code individually, to
+[regression tests](https://the-turing-way.netlify.app/reproducible-research/testing/testing-acceptance-regression.html#regression-testing)
+that run larger portions of code and just check that the changes you make to your
+code do not change previously computed results that are not supposed to change.
+For most programming languages, tools are available to run all tests with a
+single command.
+A good choice for running Python tests is [pytest](https://docs.pytest.org).
+
+## Write documentation
+It is recommended that you write documentation for your code.
+For a small piece of software, adding a README.md file with some installation
+and usage instructions might be sufficient.
+For a larger software package, having a webpage with for example usage
+instructions, examples, tutorials, contribution guidelines, and a code
+of conduct is recommended.
+This could for example be set up by using [mkdocs](https://www.mkdocs.org/).
+
+For software packages that can be [used as a library](#use-libraries), it is
+recommended that you use a tool that can build a documentation webpage from
+documentation included in the code.
+For Python projects, [sphinx](https://www.sphinx-doc.org) is a common choice.
+
+It is recommended that you
+[host your documentation on a website](#host-your-documentation-on-a-website).
+
```

Comment:
A mention here about style guides for documentation will be helpful, because I think that most people has even more doubts about how to document that about how to code. After all, documentation usually don't blow up when it is wrong.

Reply:
This has been added.